src/Time/Units.hs

```diff
@@ -312,6 +313,22 @@ fortnight = time
 floorUnit :: forall (unit :: Rat) . Time unit -> Time unit
 floorUnit = time . fromIntegral @Natural . floor
 
+{- | Convert time to the 'Num' in given units.
```

Comment:
I think it's always useful to add use cases in documentation. For example, would be good to write, that instead of writing `foo :: POSIXTime foo = 10800` one can write `foo = toNum @Second $ hour 3`. So people could learn how and why use some functions. They are exist on purporse (functions, not users).